agent/functional_tests/util/utils.go

```diff
@@ -246,14 +246,20 @@ func (agent *TestAgent) startAWSVPCTask(taskDefinition string) (*TestTask, error
 		return nil, err
 	}
 
+	securityGroup, err := getSecurityGroup()
+	if err != nil {
+		return nil, err
+	}
+
 	agent.t.Logf("Starting 'awsvpc' task in subnet: %s", subnet)
 	resp, err := ECS.StartTask(&ecs.StartTaskInput{
 		Cluster:            &agent.Cluster,
 		ContainerInstances: []*string{&agent.ContainerInstanceArn},
 		TaskDefinition:     &taskDefinition,
 		NetworkConfiguration: &ecs.NetworkConfiguration{
 			AwsvpcConfiguration: &ecs.AwsVpcConfiguration{
-				Subnets: []*string{&subnet},
+				Subnets:        []*string{&subnet},
+				SecurityGroups: []*string{&securityGroup},
```

Comment:
Do we accept the security group name here as well as the ID?  I think IMDS is returning the name.

Reply:
Looks like it doesn't accept the name, I will modify this.